packages/commonwealth/client/scripts/views/pages/QuestDetails/QuestActionCard/QuestActionCard.tsx

```diff
@@ -69,14 +70,35 @@ const QuestActionCard = ({
           {doesActionRequireCreatorReward(questAction.event_name) && (
             <CWText type="caption" className="xp-shares">
               <span className="creator-share">
-                {questAction.creator_reward_weight}%
+                {questAction.creator_reward_weight * 100}% (
+                {questAction.creator_reward_weight * 100} XP)
               </span>
-              &nbsp; shared with {actionCopies.shares[questAction.event_name]}
+              &nbsp; shared with {actionCopies.shares[questAction.event_name]}.
+              Your share ={' '}
+              {Math.abs(
+                questAction.reward_amount -
+                  questAction.creator_reward_weight * 100,
```

Comment:
questAction.creator_reward_weight * 100

this could be a const as it is used 3 times